src/components/alert/DescriptionAlert.vue

```diff
@@ -26,8 +27,6 @@ export default {
 }
 </script>
 
-<style scoped lang="scss">
-  .el-alert {
-    font-family: 'Lato';
-  }
+<style lang="scss">
```

Comment:
I think we can remove these lines. As we are not writing css at this moment for this component. What do you think?

Reply:
I'd like to leave these `<style>` sections in even when we arent using at the moment. Since we are going through multiple styling iterations, it will make it easy to add new things :)